Source/Core/Core/HW/Memmap.cpp

```diff
@@ -57,6 +58,69 @@ u8* m_pL1Cache;
 u8* m_pEXRAM;
 u8* m_pFakeVMEM;
 
+// s_ram_size is the amount allocated by the emulator, whereas s_ram_size_real
+// is what will be reported in lowmem, and thus used by emulated software.
+// Note: Writing to lowmem is done by IPL. If using retail IPL, it will
+// always be set to 24MB.
+static u32 s_ram_size_real;
+static u32 s_ram_size;
+static u32 s_ram_mask;
+static u32 s_fakevmem_size;
+static u32 s_fakevmem_mask;
+static u32 s_L1_cache_size;
+static u32 s_L1_cache_mask;
+static u32 s_io_size;
+// s_exram_size is the amount allocated by the emulator, whereas s_exram_size_real
+// is what gets used by emulated software.  This is usually 64MB.
+static u32 s_exram_size_real;
+static u32 s_exram_size;
+static u32 s_exram_mask;
+
+u32 GetRamSizeReal()
+{
+  return s_ram_size_real;
+}
+u32 GetRamSize()
```

Comment:
I'm surprised clang-format is happy with the missing line breaks between these functions... Oh well.

Reply:
I'm surprised clang-format is happy turning what was beautifully described in 11 lines into a mess that takes 44.

Reply:
Completely unnecessary response. We have code-base wide formatting to be consistent across the board and not have individual files with bespoke formatting choices that differ based on what programmer decided to code the file at that given point in time. Please direct snark like this elsewhere in the future, because it's not useful or helpful in a review.